src/main/java/core/basesyntax/dao/WarehouseDaoRead.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.service.FruitTransaction;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WarehouseDaoRead {
+    private static final String SEPARATOR = ",";
+    private static final int SKIP_LINE = 1;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QTE_INDEX = 2;
+
+    public static List<FruitTransaction> readData(String fileName) {
```

Comment:
It is recommended to make the 'readData' method an instance method instead of a static method. This will make it easier to extend or modify the class in the future.